modules/installation/pages/proxy-setup.adoc

```diff
@@ -194,46 +194,72 @@ filename: "pxelinux.0"
 If you do not use the reactivation key, you will need to re-register all the clients against the new proxy.
 
-[NOTE]
-====
-The reactivation key is only needed if you do not want to lose the history of the machine.
-If you do not use a reactivation key, the replacement proxy will become a "new" one with a new ID.
-====
 
 
-[[proxy.migration3.replace]]
 .Procedure: Replacing a Proxy and Keeping the Clients Registered
-. Before starting the actual migration procedure, save the data from the old proxy, if needed.
-    Consider copying important or manually created data to a central place that can also be accessed by the new proxy.
+. Before starting the migration, save the data from the old proxy, if needed.
+  Consider copying important or custom data to a central place that can also be accessed by the new proxy.
 . Shut down the proxy.
 . Install a new {productname} Proxy.
-    For installation instructions, see xref:install-proxy-unified.adoc[Proxy Installation].
+  For installation instructions, see xref:install-proxy-unified.adoc[Proxy Installation].
 . In the {productname} {webui}, select the newly installed {productname} Proxy, and delete it from the systems list.
 . In the {webui}, create a reactivation key for the old proxy system: On the System Details tab of the old proxy click [guimenu]``Reactivation``.
   Click [guimenu]``Generate New Key``, and make a note of the new key.
 . Register the proxy with a bootstrap script as described in xref:installation:proxy-registration.adoc[].
-  In the bootstrap script set the reactivation key as [systemitem]``ACTIVATION_KEYS``.
+  In the bootstrap script set the reactivation key with the [systemitem]``ACTIVATION_KEYS`` parameter.
   For more information about using reactivation keys, see xref:client-configuration:activation-keys.adoc[].
 
-. OPTIONAL: After the installation of the new proxy, you might also need to:
-+
+After the installation of the new proxy, you might also need to:
+
 * Copy the centrally saved data to the new proxy system
 * Install any other needed software
 * Set up TFTP synchronization if the proxy is used for autoinstallation
 
-[IMPORTANT]
-====
-During the installation of the proxy, clients will not be able to reach the {productname} Server.
-After you have deleted a proxy, the systems list can be temporarily incorrect.
-All clients that were previously connected to the proxy will show as being directly connected to the server instead.
-After the first successful operation on a client, such as execution of a remote command or installation of a package or patch, this information will automatically be corrected.
-This may take some hours.
-====
+
+
+=== Change a Proxy from Traditional to Salt
+
+You can reinstall the proxy to switch from a traditional to a Salt proxy.
+In this method, instead of a reactivation key, use the same activation key you used to register the proxy in the first instance.
```

Comment:
Maybe, I'm wrong, but I think the most important thing is to assign the same name and IP to the new proxy.  Yes, this info is provided later on, but should be mentioned here.

The expression about using the "same" activation key is a little bit ambiguous because it is not available on the newly installed proxy, and it could either mean to create an activation key with the same name/label, the same info, or both.

Reply:
^^ @jordimassaguerpla ?

Reply:
"same" activation key means reusing the one you used for the "old" proxy, so SUMA does already have it in the db. Maybe we should say "reuse" instead of "same"...

Reply:
ok, i've updated the wording slightly. LMK.